library/Fission/Environment.hs

```diff
@@ -59,3 +60,10 @@ getFlag key = do
   mayStr <- lookupEnv key
   let mayVal = truthy . fmap toLower <$> mayStr
   return mayVal
+
+-- | Get a given flag, default to the given value if not found
+getFlagWithDefault :: String -> Bool -> IO Bool
+getFlagWithDefault flag defaultValue =
+  getFlag flag >>= \case
+    Just value -> return value
+    Nothing -> return defaultValue
```

Comment:
FYI, we have acombinator for fallback values:

```haskell
getFlag flag .!~ fallback
```

Reply:
If you don't like operators in particular, we can do a named version, but let's keep these orthogonal:

```haskell
fallback :: Monad m => m (Maybe a) -> a -> m a
fallback = (.!~)
-- AKA: fallback mayValue defValue = pure (fromMaybe defValue) <*> mayValue

"TIMEOUT"
|> getFlag
|> fallback 42
```